appveyor.yml

```diff
@@ -125,11 +125,11 @@ build_script:
 
     docker-switch-linux
 
-    # docker run -d --name pgregress -v c:/projects/timescaledb:/timescale timescaledev/postgresdev:exec_backend-10-alpine
+    docker run -d --name pgregress alpine:3.12 sh -c "while sleep 3600; do :; done"

```

Comment:
you wanna use alpine:3.8 here to get postgresql 10 to match the rest of appveyor config or adjust rest of appveyor to use postgresql 12